drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
```

Comment:
I removed that since it's up to whoever implements the actual remote drivers (i.e., another PR). But I can restore the field to the struct, and return it in `Type`, if you prefer.